### ISSUE TITLE
Is it possible to use KISS on a test image?

### ISSUE THREAD

**Issue by Horace89** [Bartzi/kiss#5]
Is there a way to test KISS on a single image? Ex:
```
python recognize.py ./image1.py
```
which would return the recognized text in image1?

**Comment by Bartzi** [comment]
No, there is currently no code available that supports this, but you can create a script for that if you like :wink:

**Comment by Horace89** [comment]
Ok thanks!

**Comment by ank127** [comment]
Do we need char_map for evaluating KISS on a single image?

**Comment by Bartzi** [comment]
Yes, you will need the char_map in order to map the predicted classes back to characters.

**Comment by ank127** [comment]
Thanks a lot for the reply.
For creating  testing script we need char_map dictionary. 
If possible, can you please provide the char_map  of your trained model in model.zip

**Comment by Bartzi** [comment]
You can find the `char_map` [here](https://github.com/Bartzi/kiss/blob/master/train_utils/char-map-bos.json) :smiley:

**Comment by ank127** [comment]
Thanks :):)

**Comment by desean1625** [comment]
@Horace89 Did you come up with a script for this? Is so please share so.

**Comment by ank127** [comment]
Everything is available on git hub code. You need to dig a bit. Currently i dont have the code else would have provided. It took sometime to create that script

**Comment by fredO13** [comment]
at least it would be nice to document the arguments for the evaluate.py script a bit more... for example it is not obvious what is expected as _**snapshot-prefix**_.
Thanks

**Comment by Clement13430** [comment]
Yes, I agree, I did not understand as well what is the meaning of **snapshot-prefix** and what are we supposed to put in this argument,
If you can help me a bit, that would be great,

Thank you by advance
